### DISCUSSION TITLE
Feature Request: "Pending" and "Installing" before "Installed"

### DISCUSSION THREAD

**Discussion by kentprimrose** [mason-org/mason.nvim#700] [General]
It would be great to have the "Pending" and "Installing" sections come before "Installed". That was we could easily see the status when updating plugins. Currently we have to scroll to the end of the "Installed" list to tell when updates are complete. Having the "Pending" and "Installing" sections come first would show the progress of updating, and would show it's completion when they go away.

**Comment by williamboman** [comment]
I think that could make sense! I remember the rationale behind the current placement was to avoid jittering in the UI by moving content that changes frequently further down the buffer. I'll play around with it!

**Comment by williamboman** [comment]
Try pulling the latest plugin version @kentprimrose. I did some improvements (I hope) in https://github.com/williamboman/mason.nvim/pull/712.

**Comment by kentprimrose** [comment]
Excellent! Very useful and works great. Thanks!

On Thu, Dec 1, 2022 at 1:37 PM William Boman ***@***.***>
wrote:

> Try pulling the latest plugin version @kentprimrose
> <https://github.com/kentprimrose>. I did some improvements (I hope) in
> #712 <https://github.com/williamboman/mason.nvim/pull/712>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/williamboman/mason.nvim/discussions/700#discussioncomment-4286009>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALPZKKF4T2RKH3AXXFFJIDWLDV45ANCNFSM6AAAAAASMZEXNY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Kent Primrose
http://www.linkedin.com/in/kentprimrose
